plyer/platforms/android/qrbarcodereader.py

```diff
@@ -0,0 +1,21 @@
+from jnius import cast
+from jnius import autoclass
+from plyer.facades import QrBarcodeReader
+
+PythonActivity = autoclass('org.renpy.android.PythonActivity')
+Intent = autoclass('android.content.Intent')
+Uri = autoclass('android.net.Uri')
+
+class AndroidQrBarcodeReader(QrBarcodeReader):
+
+    def _scan(self):
+
+        intent = Intent()
+        intent.setAction(Intent.ACTION_VIEW)
+        intent.setData(Uri.parse('http://www.onlinebarcodereader.com'))
```

Comment:
@tito  i will try native/library-based implementation.It will take little time to implement as i have to research more ways to achieve it on other platforms  but yes  i will do it.I think from ios7 there is inbuilt feature to establish it.